src/plugins/vuetify.ts

```diff
@@ -0,0 +1,12 @@
+import Vuetify from 'vuetify';
+import 'vuetify/dist/vuetify.min.css';
+import '@mdi/font/css/materialdesignicons.min.css';
+
+export default (Vue): typeof Vuetify => {
```

Comment:
Why did you choose this way of writing instead of : 
```
Vue.use(Vuetify)

export default new Vuetify({ ... })
```

(I looked at the documentation of vuetify => https://vuetifyjs.com/en/getting-started/quick-start#full-installation

Reply:
It was to ensure that Vuetify use the same Vue instance as we do. This doesn't seams necessary after all, let's remove it.